python/pyspark/pandas/generic.py

```diff
@@ -3249,7 +3250,7 @@ def interpolate(
 
         Parameters
         ----------
-        method : str, default 'linear'
+        method : str, default None
```

Comment:
Would you mind giving some context why this is changed while pandas is still using 'linear' as default ??

Reply:
my bad, will make it keep in line with pandas